src/Components/Sidebar/Sidebar.tsx

```diff
@@ -29,29 +35,23 @@ const Sidebar = ({ account }) => {
           <div className={styles.language}>
```

Comment:
<div className={`${styles.language} ${isSelcted? styles.selcted : "" }`} />

if selected => .language.selected
 not => .language